Gems/MotionMatching/Code/Source/FeatureMatrixTransformer.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/std/containers/span.h>
+#include <Allocators.h>
+#include <FeatureMatrix.h>
+
+namespace EMotionFX::MotionMatching
+{
+    //! Transformers can be used to e.g. normalize or scale features in the feature matrix or the query vector.
+    class FeatureMatrixTransformer
+    {
+    public:
+        AZ_RTTI(Transformer, "{B19CDBB8-FA99-4CBD-86C1-640A3CC5988A}");
+        AZ_CLASS_ALLOCATOR(FeatureMatrixTransformer, MotionMatchAllocator, 0)
```

Comment:
nitpick: This is missing a semicolon at the end of the line.

Reply:
Changed